README.md

```diff
@@ -25,8 +25,7 @@ This plugin uses a custom build tool called [`sake`](https://github.com/skyverge
 
 If you have `sake` set up on your system, these commands can be used to generate a production build.
 
-- `npx sake build` builds to `/build/facebook-for-woocommerce`.
```

Comment:
I'm not sure if it's worth setting up a script for this also?

Reply:
Yes I think it's worth it - `package.json` is a good place to document the essential workflow scripts (alongside actual docs). Note this may still need a global install of `sake` on dev machine. Feel free to add all these while you're there, and update readme!

Reply:
What's the purpose of having a script that build the plugin into a different directory though? This may encourage people to manually ZIP it when doing releases which we should discourage because it often results in unwanted (.DS_Store) files ending up in the release. Not a big issue though since it's listed right next to `npx sake zip`.

Reply:
Agreed, the zip command is the more important one. I forget why I documented `sake build` specifically, maybe just to clarify that it _isn't_  the primary build command!?

Reply:
I think I remember now - when you check out this repo, you need to build the code somehow. This was my stab at documenting how to do that with sake (without deploying!). This will change when the PR merges - feel free to remove `npx sake build` if it's not a core dev flow :)

Reply:
Ok cool. After this PR `npm start` should be used which you have already documented in the README.md 😄

Reply:
`npm start` .. ? never heard of it 🤣